### DISCUSSION TITLE
VcAnalyses的可视域分析功能

### DISCUSSION THREAD

**Discussion by oneya999** [zouyaoji/vue-cesium#457] [Q&A]
您好，我在[vue-cesium官方示例](https://vue-cesium.songluck.com/#/zh-CN/component/analyses/vc-analyses)上的VcAnalyses模块中使用了可视域分析功能，在可视域绘制结束之后，使用鼠标滚轮缩放地球时，可视域面的颜色会发生一些变化。请问这是glsl着色器的问题吗还是cesium shadowmap的原因？想请教一下大佬有没有啥方法可以固定这个可视域面，使之在缩放地球时不发生变化？
![1](https://github.com/zouyaoji/vue-cesium/assets/67720922/02d8b79e-f480-4263-a0df-32533aa6e0c4)
![2](https://github.com/zouyaoji/vue-cesium/assets/67720922/43f3c35e-63d3-4f03-bc11-59b285000527)

**Comment by zouyaoji** [comment]
没有哈。

**Reply by houluy** [reply]
请问这是什么原因导致的？
